src/app/component/eco-news/news-preview-page/news-preview-page.component.html

```diff
@@ -14,24 +14,31 @@
         </div>
       </a>
     </div>
-    <div class="publish">
-      <div>Publish</div>
-    </div>
+    <button type="submit"
+            [disabled]="!validData"
```

Comment:
Please, remove it 'cause the Publish button should be unavailable (in case of invalid data) on CreateNews Component, but not here

Reply:
fixed